stories/index.tsx

```diff
@@ -44,8 +44,11 @@ require('../src/components/NewCard/docs/index.story');
 require('../src/components/SocialBar/docs/index.story');
 require('../src/components/ShareButton/docs/index.story');
 require('../src/components/CopyUrlButton/docs/index.story');
+require('../src/components/RadioButton/docs/index.story');
 require('../src/components/Picker/docs/index.story');
 require('../src/components/CalendarCell/docs/index.story');
 require('../src/components/Event/docs/index.story');
 require('../src/components/CalendarPopover/docs/index.story');
 require('../src/components/Spinner/docs/index.story');
+require('../src/components/RadioButtonGroup/docs/index.story');
+require('../src/components/Spinner/docs/index.story');
```

Comment:
The last line was already added, so I don't understand why it shows it's new here.
You probably need to rebase